addons/SimpleTODO/TODO.gd

```diff
@@ -1,16 +1,31 @@
 tool
-extends VBoxContainer
+extends Control
 
-onready var column_container = $ScrollContainer/Columns
+onready var column_container = $VBoxContainer/ScrollContainer/Columns
+onready var vbox_container = $VBoxContainer
 
 var undo_redo: UndoRedo
 
+onready var item_placement_holder_scene = preload("res://addons/SimpleTODO/ItemPlacementHolder.tscn")
```

Comment:
Doesn't need to be onready.

Reply:
So preload isn't required at all? Then what's the point of preload then? I'm not sure if I'm getting this right.

Reply:
Preload will load the resource together with the script, so every time this line is executed, it will use a cached resource instead of loading it each time. It's the same as loading a resource and storing in a variable, so you can use it directly instead of a variable. Variables are mostly useful for arrays of resources or when you need to use `load()` instead (e.g. when path is not constant).